test/e2e/helpers.js

```diff
@@ -443,85 +443,6 @@ const openDapp = async (driver, contract = null, dappURL = DAPP_URL) => {
     ? await driver.openNewPage(`${dappURL}/?contract=${contract}`)
     : await driver.openNewPage(dappURL);
 };
-const STALELIST_URL =
```

Comment:
Thanks for removing these mocks from `test/e2e/helpers.js`. 
Ideally, mocks should live in `test/e2e/mock-e2e.js` or with the tests.